enhancements/platforms/ovirt-rhv-ipi.md

```diff
@@ -0,0 +1,271 @@
+---
+title: ovirt-rhv-platform-provider
+authors:
+  - "@rgolangh"
+reviewers:
+  - "@deads2k"
+  - "@crawford"
+  - "@abhinavdahiya"
+  - "@sdodson"
+approvers:
+  - "@deads2k"
+creation-date: 22019-10-07
+last-updated: 2019-10-07
+status: implementable
+---
+
+# ovirt-rhv-platform-provider
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Open Questions [optional]
+
+## Summary
+
+This document describes how `oVirt` becomes a platform provider for Openshift. \
+`oVirt` is a virtualization platform and is similar to the `baremetal` platform \
+ provider it is lacking DNS and Load-Balancing services, but it has the advantage \
+if software-defining your datacenter, utilizing existing hardware and making \ 
+that pain free and fast.
+Like `baremetal` platform it uses the [OpenShift-Metal³ kni-installer](https://github.com/openshift-metal3/kni-installer) method \
+- essentially providing an internal cluster-level DNS service using mDNS and
+ coreDNS, and load-balancing using keepalived.
+
+Components involved:
```

Comment:
what about the ingress-controller?

Reply:
I'm adding a note to reference this in baremetal networking doc. 
Also, I don't think I need to change something in ingress controller itself but
I do need t ask the user for the ingress IP and hence I added this in the PR the `PlatformStatus`:
```
type OvirtPlatformStatus struct {
	...

	// ingressIP is an external IP which routes to the default ingress controller.
	// The IP is a suitable target of a wildcard DNS record used to resolve default route host names.
	IngressIP string `json:"ingressIP,omitempty"`
}
```